### ISSUE TITLE
Chapter11 Workout App crashes after selecting a workout

### ISSUE THREAD

**Issue by williamspete321** [dogriffiths/HeadFirstAndroid#30]
The workout list appears when I open the app, but the app crashes when I select a specific workout. I have followed the code exactly as it is in the 2nd edition book from Chapter 11. The app worked fine in Chapter 10, it's just not working after making changes for the stopwatch. I also am not getting any error messages when it crashes.  I am using Android Studio 3.3.1. Any idea what could be going on?

**Comment by dogriffiths** [comment]
Hello William,

Sorry you’re having problems. Have you tried it with the code from the 2nd edition repo?

https://github.com/dogriffiths/HeadFirstAndroid2ndEdition

DG

**Comment by williamspete321** [comment]
Hi DG,

Yes I used the code from the 2nd edition book and double checked with the code posted here.

**Comment by dogriffiths** [comment]
Hello William,

Apologies for the delay in replying. I’ve tried to reproduce the issue, but I’ve not been able to. Would it be possible to capture the error from “Logcat” in Android Studio. Typically a crashed app will generate a stack trace in the LogCat tab which will show the error. That should tell us what cause the crash.

Thanks,

DG

**Comment by williamspete321** [comment]
<img width="1235" alt="Screen Shot 2019-03-11 at 10 31 37 AM" src="https://user-images.githubusercontent.com/34874236/54131575-fdbce700-43e8-11e9-9850-4272796bf6a6.png">

I found my error! It was a simple error; I had left out a "d" when I copied the code in beforehand, and I still missed this even when I checked the code multiple times against the book and your code here.

I'm sorry if I wasted your time. I should have seen this earlier in Logcat. Usually I see Logcat when there's an error, but I thought there was no error when the app crashed. I've updated AndroidStudio a few times and perhaps in the past it automatically jumped from Build to Logcat. Though this time, everything is built correctly, and it just crashed when I selected a workout.

I'm still curious why this error caused the whole app to crash? Anyway, thank you again David for taking the time to help. I should have caught this if I was more familiar with Logcat, Build, etc.
